com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformTests.cs

```diff
@@ -588,6 +588,7 @@ private void AllChildrenLocalTransformValuesMatch(bool useSubChild)
             success = WaitForConditionOrTimeOutWithTimeTravel(AllChildObjectInstancesHaveChild);
             Assert.True(success, "Timed out waiting for all instances to have parented a child!");
 
```

Comment:
Both minor tweaks to this test will help prevent random edge case precision validation errors.